### DISCUSSION TITLE
Training and Validation Errors

### DISCUSSION THREAD

**Discussion by tjwolter** [mir-group/nequip#329] [Q&A] [answered]
I have noticed that my training set errors are consistently larger than my validation set errors when I train a model. Is this due to the model not re-scaling into real units, similar to the answer to another question about the difference between model.train() and model.eval()?

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @tjwolter ,

I'm struggling right now to find the relevant GitHub discussion, but maybe you'll have better luck than me:   this is a consequence of our use, by default, of EMA averaging on the validation model (but not the training model) and is generally expected.  All reported _errors_ are in real units, regardless.  (Losses I think it depends, I'd have to check again and remind myself.)
